docs/source/hardware/lowpriority.rst

```diff
@@ -0,0 +1,75 @@
+
+.. _partition_names:
+
+**********
+Low priority partition
+**********
+:sub:`Updated September, 2022`
+
+
+Introduction
+===================
+``lowpriority`` is a new partition on our high performance computing (HPC) cluster - Discovery, that allows the research community to use private resources on the HPC cluster when they are idle. 
+This new partition has hardware that is not otherwise available to the general research community and, in time, could double the resources available to NURC users. 
+This is a common practice in HPC clusters to optimize the use of idle private resources that consume power and cooling. 
+
+When to use the low priority partition
+===================
+
+From most to least-recommended scenarios:
+
+1. Code that can be `checkpointed <https://rc-docs.northeastern.edu/en/latest/best-practices/checkpointing.html?highlight=array#gromacs-checkpointing-example>`_
+2. Jobs that fit on a single node
+3. Jobs that require multiple nodes (eg, `MPI <https://northeastern-university-rc-public-documentation--19.com.readthedocs.build/en/19/software/mpi.html>`_)
```

Comment:
the 'eg,' should be changed to, 'e.g., '

Reply:
Done